### PR TITLE
Ksyed245 : added in h1 ksyed245

### PR DESCRIPTION
**Pull request by khurram-syed**
Adding a pull request for kasyed245

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body id="gradient">
-	<h1>Background Generator- Branch Master</h1>
+	<h1>Background Generator- Branch Master - KASYED245</h1>
```
Comment (khurram-syed) [inline index.html:8]: Looks fine.

**Comment by kasyed245** [comment]
Kasyed245 : just added my name in h1 header

File: index.html (hunk 1 above)
Comment (kasyed245) [inline index.html:8]: kasyed245: here where I did my changes

**Review by khurram-syed — COMMENTED** [review]
khurram-syed : all looks good

**Pull request merged**